net/squid/patches/010-no-buildbxxflags.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/Makefile.am
```

Comment:
Was this patch sent to upstream? There isn't a commit author, commit description.

Reply:
I don't think it's appropriate for upstream as this is specific to OpenWrt.

Reply:
Hm, but this should be noted or written in this file, what it does, why it is needed. We need to have a clean and proper history.